File: public/javascripts/mobile/app/model/EligibleReward.js
Ext.define('Genesis.model.EligibleReward',
{
   extend : 'Ext.data.Model',
   id : 'EligibleReward',
   alternateClassName : 'EligibleReward',
   config :
   {
      idProperty : 'id',
      fields : ['id', 'reward_id', 'reward_title', 'reward_text', 'reward_type', 'photo']
   }
});

File: app/models/eligible_reward.rb
class EligibleReward
  attr_accessor :reward_id, :reward_type, :reward_title, :reward_text
  
  def initialize(reward_id, reward_type, reward_title, reward_text)  
    @reward_id = reward_id
    @reward_type = reward_type
    @reward_title = reward_title
    @reward_text = reward_text
  end
end